pages/cloudflare/howtos/keep_names.mdx

```diff
@@ -0,0 +1,38 @@
+import { Callout } from "nextra/components";
+
+## `__name` issues
+
+When using the OpenNext adapter, Wrangler processes the worker's code with [esbuild](https://esbuild.github.io/), and by default
+enables the [`keep-names`](https://esbuild.github.io/api/#keep-names) option. While this is generally useful for debugging, it can
+cause issues with certain Next.js libraries (e.g. [next-themes](https://www.npmjs.com/package/next-themes)) that convert scripts
+to strings. This happens because `esbuild` introduces an `__name` function at the top of modules, which may inadvertently appear
+in the generated script strings. When these strings are evaluated at runtime, the `__name` function might therefore not be defined,
+leading to errors like the following:
+
+```
+Uncaught ReferenceError: __name is not defined
+```
+
+<Callout>
+  Note that depending on your minification settings, the `__name` identifier might be minified, making the
+  error message less clear and potentially not explicitly mentioning `__name` in such cases.
+</Callout>
+
+### How to fix such issues
+
+To fix the issue you can simply set the `keep_names` option to `false` in your `wrangler.jsonc` file, like in the following example:
```

Comment:
Too bad we don't have direct links to each option, but still the following could be nice

```suggestion
To fix the issue you can simply set the `keep_names` option to `false` in your [`wrangler.jsonc` file](https://developers.cloudflare.com/workers/wrangler/configuration), like in the following example:
```

Reply:
> Too bad we don't have direct links to each option

yeah I was thinking the exact same thing 😓 
(well it shouldn't be too problematic to do, maybe we can do it as some point as a quick win 🤔)